org.eclipse.xtext.xbase/src/org/eclipse/xtext/xbase/annotations/typing/XAnnotationUtil.java

```diff
@@ -70,5 +73,21 @@ public XExpression findValue(XAnnotation annotation, JvmOperation jvmOperation)
 		}
 		return emptySet();
 	}
-	
+
+	/**
+	 * @since 2.15
```

Comment:
Please add Javadoc, e.g. the annotations are returned in order of their appearance.